### DISCUSSION TITLE
这个项目会考虑支持SRT协议吗？ / Will this project consider supporting the SRT protocol?

### DISCUSSION THREAD

**Discussion by Mishasama** [sorayuki/obs-multi-rtmp#359] [Q&A] [answered]
如题。 / As the title.

**Comment by lucaspontoexe** [comment, marked as answer]
Check my pull request (#403)  - I found what needed to be changed in order to allow SRT publishing. 
There are plugin binaries in my forked repository: https://github.com/lucaspontoexe/obs-multi-rtmp/actions/runs/7856423201
